backend/src/backend/constants.py

```diff
@@ -9,17 +9,17 @@
 
 COLLECTION_EXPIRE_AFTER = datetime.timedelta(days=7)
 
-if not os.getenv("POSTGRES_URI"):
-    raise EnvironmentError("Please set the POSTGRES_URI environment variable")
-else:
-    POSTGRES_URI = os.getenv("POSTGRES_URI")
 
 
 @dataclass
 class BackendConf:
     """
     Backend configuration, read from environment variables and set default values.
     """
+    if not os.getenv("POSTGRES_URI"):
+        raise EnvironmentError("Please set the POSTGRES_URI environment variable")
+    else:
+        POSTGRES_URI = os.getenv("POSTGRES_URI")
```

Comment:
Again, use `postgres_uri` (lower case). Consistency is important and PEP8 attributes meaning to case.